modules/apps/object/object-service/src/main/java/com/liferay/object/internal/bulk/selection/DefaultPermissionObjectBulkSelectionAction.java

```diff
@@ -145,6 +193,32 @@ private long _getObjectDefinitionId(long companyId) throws PortalException {
 		return objectDefinition.getObjectDefinitionId();
 	}
 
+	private JSONObject _mergeDefaultPermissionsForAssetType(
+		JSONObject existingDefaultPermissionsJSONObject,
```

Comment:
just existingJSONObject and newJSONObject
and assetType -> key

(than please sort)